src/components/OnboardingLoader.js

```diff
@@ -0,0 +1,28 @@
+import React, { useCallback, useState } from 'react'
+import OnboardingModal from './Onboarding/OnboardingModal'
+
+function OnboardingLoader({ children }) {
+  const [onboardingCompleted, setOnboardingCompleted] = useState(
+    localStorage.getItem('onboardingCompleted')
+  )
+
+  const handleStartCourt = useCallback(() => {
+    localStorage.setItem('onboardingCompleted', true)
+    setOnboardingCompleted(true)
+  }, [])
+
+  return (
+    <React.Fragment>
+      {!onboardingCompleted && (
+        <OnboardingModal
+          visible={!onboardingCompleted}
+          onStartCourt={handleStartCourt}
```

Comment:
What do you think about renaming this (and its handler) to `onOnboardingComplete()`?

Reply:
I think we could even go for `onComplete` since we are in the context of the `OnboardingModal`

Reply:
Done (using `onComplete`) https://github.com/aragon/court-dashboard/pull/76/commits/785a341a70f9d0d380e4e831d058c9cc9b1006f0